src/cmd/internal.go

```diff
@@ -156,12 +157,30 @@ tableOfContents: false
 	},
 }
 
+func addGoComments(reflector *jsonschema.Reflector) error {
+	addCommentErr := errors.New("this command must be called from the root of the Zarf repo")
+
+	typePackagePath := filepath.Join("src", "types")
+	if err := reflector.AddGoComments("github.com/zarf-dev/zarf", typePackagePath); err != nil {
+		return fmt.Errorf("%w: %w", addCommentErr, err)
+	}
+	varPackagePath := filepath.Join("src", "pkg", "variables")
+	if err := reflector.AddGoComments("github.com/zarf-dev/zarf", varPackagePath); err != nil {
+		return fmt.Errorf("%w: %w", addCommentErr, err)
+	}
```

Comment:
I am always a bit worried when I see use of reflections. Could we get an explanation what this is actually doing?

Reply:
I'm pretty sure it's literally parsing the file rather than using reflections. https://github.com/invopop/jsonschema?tab=readme-ov-file#using-go-comments